pkg/interfaces/contracts/liquidity-mining/IStakelessGauge.sol

```diff
@@ -17,7 +17,19 @@ pragma solidity >=0.7.0 <0.9.0;
 import "./ILiquidityGauge.sol";
 
 interface IStakelessGauge is ILiquidityGauge {
+    /**
+     * @dev Performs a checkpoint, computing how much rewards should be minted for the gauge.
+     */
     function checkpoint() external payable returns (bool);
 
+    /**
+     * @dev Returns the address that will receive the rewards (either the L2 gauge, or a mainnet address).
+     */
     function getRecipient() external view returns (address);
+
+    /**
+     * @dev Returns total ETH bridge cost in wei.
+     * Each `checkpoint` should receive this exact amount to work.
```

Comment:
Mention that some networks will need this and some won't? (Maybe with a list of the current ones who do?)

Reply:
+1.
I'd just say that in some cases the returned value will be 0.

The current checkpointer won't be compatible with this gauge; we knew this would happen at some point.

However, so far we have two cases:
- Either you forward 0 value.
- You query how much to send (somehow) and you send that amount.

We can make the gauges comply with this interface so that all gauges from now on.
For old gauges that don't have the function, the checkpointer can just try / catch and make the cost zero if the call fails. That should be enough to make the checkpointer even more future proof.